tests/parse_many_test.cpp

```diff
@@ -101,14 +101,6 @@ bool validate(const char *dirname) {
                 printf("size of file in bytes: %zu \n", json.size());
                 everything_fine = false;
             }
-            if(!error) {
```

Comment:
I understand why we might want to remove the line "auto iter = simdjson::document::iterator(doc);", but removing the parser.load_many test seems like it removes our test. It is fine if we have others tests for load_many, and it seems that we do in tests/errortests.cpp but I'd like you to confirm it.

Reply:
Yep, I removed it because I had added positive load_many/parse_many tests to basictests when I fixed that bug.